### DISCUSSION TITLE
hide or disable the data reminder box

### DISCUSSION THREAD

**Discussion by CACG1022** [opengisch/QField#5249] [Bugs]
Is there any way to ensure that hidden fields are left empty even when the data autofill option is selected? Currently, we are having issues because the project contains fields that are shown or hidden depending on the value of another field. The problem occurs when we need to fill in new data and the autofill option remains selected for a hidden field. Is there also a way to disable the autofill option for certain fields?
![ayuda checkbox](https://github.com/opengisch/QField/assets/118937011/5154994e-44a6-4832-bbe6-1e9b2b65b591)

**Comment by SeqLaz** [comment]
@CACG1022 If the field has default values you can use the Widget Type of Hidden

![image](https://github.com/opengisch/QField/assets/77129293/feea305b-f1c6-4fb4-87a3-3dec4fcefcbc)

**Reply by CACG1022** [reply]
Hi. @SeqLaz  I don't need to hide the field; I need to hide the reminder box. That is, to prevent the person filling out the field from repeating the information.

**Comment by suricactus** [comment]
@CACG1022 unfortunately at the moment there is no way to hide the checkbox for the "remember value" functionality. Note that recent development made the checkboxes appear as a distinct icon instead of an alien square:
![image](https://github.com/opengisch/QField/assets/2820439/d1d24817-38ab-4edc-a480-7e14bc46994b)

There is some planned work to further integrate the "remeber value" better in the future into a context menu for each field, but it is still seeking the funding. If you are interested, please contact us at sales@qfield.org .

**Comment by RatInTheField** [comment]
I too would like there to be an option to hide the remember value box/pin or to make it inactive when the field is not visible on the form. In my experience, having the remember value box/pin active, especially if the field is hidden from view allows inappropriate values to be entered without the user being aware.

**Comment by nirvn** [comment]
@RatInTheField , this is coming in QField 4.1 :partying_face:

**Reply by RatInTheField** [reply]
Thanks. Can you estimate when 4.1 will be released?

**Reply by nirvn** [reply]
@RatInTheField , a little after mid March (next month).

**Reply by RatInTheField** [reply]
Very good.  Thanks very much. That will save me a lot of work and hair loss!
